streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -196,6 +197,12 @@ public void configure(final Map<String, ?> configs) {
         final LogContext logContext = new LogContext(logPrefix);
         log = logContext.logger(getClass());
 
+        final String upgradeMode = streamsConfig.getString(StreamsConfig.UPGRADE_MODE_CONFIG);
+        if (StreamsConfig.UPGRADE_FROM_0100X.equals(upgradeMode)) {
+            log.debug("Downgrading to metadata version 1 for upgrade from 0.10.0.x.");
```

Comment:
Since upgrading won't be a common occurrence should this be `log.info`?  Just a thought I don't have a strong opinion either way.

Reply:
I am fine either way -- put as debug because it's usually not helpful information for the user.